src/OpenStack.Net/OpenStack/Services/ContentDelivery/V1/ServiceOriginRule.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace OpenStack.Services.ContentDelivery.V1
+{
+    using Newtonsoft.Json;
+    using OpenStack.ObjectModel;
+
+    public class ServiceOriginRule : ExtensibleJsonObject
```

Comment:
:exclamation: This needs to be documented (along with several other elements in the same file).